src/services/jsTyping.ts

```diff
@@ -46,8 +55,44 @@ namespace ts.JsTyping {
             }
         }
         
-        let newTypingNames = typingNames.filter(t => cachedTypingNames.indexOf(t) < 0);
-        
+        let newTypingNames = ts.filter(typingNames, t => cachedTypingNames.indexOf(t) < 0);
         return { cachedTypingPaths, newTypings: newTypingNames };
     }
+
+    export function updateTypingsConfig(

```

Comment:
Can you add comments of what exactly does this updating process do? Like wiping out the "not found" field, reset the "last updated" field etc.